### PR TITLE
Remove toolkit and template dependency

### PR DESCRIPTION
**Pull request by igloosi**
- Updates layouts so they use the bollerplate template from GOV.UK Frontend
- Implements Footer component from GOV.UK Frontend
- Implements Header component from GOV.UK Frontend
- Reimplements missing unbranded template things and fixes: https://github.com/alphagov/govuk-prototype-kit-private-beta/issues/41
- Removes references to Frontend toolkit and Template
- Updates server.js

### CODE REVIEW

File: app/views/layout.html [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+{% from "./components/back-link/macro.njk"     import govukBackLink %}
+{% from "./components/breadcrumbs/macro.njk"   import govukBreadcrumbs %}
+{% from "./components/button/macro.njk"        import govukButton %}
+{% from "./components/checkboxes/macro.njk"    import govukCheckboxes %}
+{% from "./components/date-input/macro.njk"    import govukDateInput %}
+{% from "./components/details/macro.njk"       import govukDetails %}
+{% from "./components/error-message/macro.njk" import govukErrorMessage %}
+{% from "./components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "./components/fieldset/macro.njk"      import govukFieldset %}
+{% from "./components/file-upload/macro.njk"   import govukFileUpload %}
```
Comment (nickcolley) [inline app/views/layout.html]: I think this list is out of date now

Comment (igloosi) [reply app/views/layout.html]: why we should have all.njk :)

File: server.js [hunk 2]
```diff
@@ -62,9 +62,12 @@ if (env === 'production' && useAuth === 'true') {
 }
 
 // Set up App
-var appViews = [path.join(__dirname, '/app/views/'),
+var appViews = [
+  path.join(__dirname, '/app/views/'),
   path.join(__dirname, '/lib/'),
-  path.join(__dirname, '/node_modules/@govuk-frontend/frontend/components')]
+  path.join(__dirname, '/node_modules/@govuk-frontend/frontend'), // template path
+  path.join(__dirname, '/node_modules/@govuk-frontend/frontend/components')
```
Comment (alex-ju) [inline server.js]: If we add only `/node_modules/@govuk-frontend/frontend` means we'll have to add `/components` to each component import. Not sure what's best.

File: docs/views/layout_unbranded.html [hunk 3]
```diff
@@ -1,9 +1,13 @@
-{% extends "govuk_template_unbranded.html" %}
+{% extends "template_unbranded.html" %}
```
Comment (alex-ju) [inline docs/views/layout_unbranded.html]: Should we change the extension of `template_unbranded.html` to `.njk` since it has Nunjucks syntax inside

**Comment by igloosi** [comment]
updated with comments

**Pull request merged**